priam/src/main/java/com/netflix/priam/config/CassandraConfiguration.java

```diff
@@ -178,6 +178,18 @@
     @JsonProperty
     private Map<String, String> extraConfigParams;
 
+    // dual account is disabled by default.
+    @JsonProperty
+    private boolean dualAccountEnabled = false;
+
+    // Amazon Resource Name (ARN) for VPC.
+    @JsonProperty
+    private String vpcEC2RoleAssumptionArn;
+
+    // SimpleDB Assume Role is false by default;
+    @JsonProperty
+    private boolean sdbAssumeRole = false;
```

Comment:
I think we only need one new parameter, `sdbRoleAssumptionArn`.  If it's present use the STS assume role credentials provider for SDB, otherwise use the default.